va_explorer/va_data_management/management/commands/dhis.py

```diff
@@ -0,0 +1,497 @@
+"""
```

Comment:
Seems to have similarly been copied from openva_pipeline. Consider removing.

Reply:
done